client/src/components/AddEventCard.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const Card = styled.form`
+  width: 280px;
+  height: 400px;
+  background: ${({ theme }) => theme.colors.cardBackground};
```

Comment:
`${theme => theme.colors.cardBackground}` would work just fine I'd say

Reply:
i tried it but theme is undefined. 
I have to destructure the props in {theme }